### DISCUSSION TITLE
Edit Entry Type button on entry page

### DISCUSSION THREAD

**Discussion by knynkwl** [craftcms/cms#10112] [Ideas] [answered]
There are a lot of times I go to an entry and realize that it's missing a field, or need to update some verbiage. 

Instead of having to go to settings, click sections, and find the entry type.  It would be a huge time saver to have a button to edit the entry type on the entry edit page.

This should only be available in dev mode/project config is editable.

**Comment by mmikkel** [comment]
FWIW the [CP Field Inspect plugin](https://plugins.craftcms.com/cp-field-inspect) adds a button just like that, but I 100% agree that a similar feature should be in core.

**Comment by brandonkelly** [comment, marked as answer]
Added for Craft 5.8! a8bfc9024e70e527d46801825899cfced3d764f3
